prime-router/src/main/kotlin/encoding/FHIR.kt

```diff
@@ -0,0 +1,74 @@
+package gov.cdc.prime.router.encoding
+
+import ca.uhn.hl7v2.model.Message
+import ca.uhn.hl7v2.model.v251.segment.MSH
+import io.github.linuxforhealth.fhir.FHIRContext
+import io.github.linuxforhealth.hl7.ConverterOptions
+import io.github.linuxforhealth.hl7.message.HL7MessageEngine
+import io.github.linuxforhealth.hl7.resource.ResourceReader
+import org.apache.logging.log4j.kotlin.Logging
+import org.hl7.fhir.r4.model.Bundle
+
+fun Bundle.encode(): String {
+    return FHIR.encode(this)
+}
+
+class FHIR {
```

Comment:
Change this to an object class.

Reply:
Instead of putting everything into the companion object, you can create this as an object class which enforces its "singleton-ness", if that really is your intention.